pkg/controller/envoy/ads_event_test.go

```diff
@@ -452,7 +456,37 @@ func TestHandleEdsResponse(t *testing.T) {
 	})
 }
 
+func initBpfMap(t *testing.T) {
+	_ = os.MkdirAll("/mnt/kmesh_cgroup2", 0755)
```

Comment:
Just ignore the return value? No error handling? Or is it because if the path creation fails, the corresponding error will be reflected in the mount handling.